clients/admin-ui/src/pages/dataset/new/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  Box,
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Heading,
+  Stack,
+  Text,
+} from "@fidesui/react";
+import type { NextPage } from "next";
+import NextLink from "next/link";
+import { useState } from "react";
+
+import Layout from "~/features/common/Layout";
+import DatasetYamlForm from "~/features/dataset/DatasetYamlForm";
+
+const NewDataset: NextPage = () => {
+  const [generateMethod, setGenerateMethod] = useState<
+    "yaml" | "database" | "manual" | null
+  >(null);
+  return (
+    <Layout title="Datasets">
+      <Heading mb={2} fontSize="2xl" fontWeight="semibold">
+        Datasets
+      </Heading>
+      <Box mb={8}>
+        <Breadcrumb fontWeight="medium" fontSize="sm" color="gray.600">
+          <BreadcrumbItem>
+            <NextLink href="/dataset">Datasets</NextLink>
+          </BreadcrumbItem>
+          <BreadcrumbItem>
+            <NextLink href="#">Create new dataset</NextLink>
+          </BreadcrumbItem>
+        </Breadcrumb>
+      </Box>
+      <Stack spacing={8}>
+        <Box w={["100%", "100%", "50%"]}>
```

Comment:
Instead of specifying 100% twice you can use the [object syntax](https://chakra-ui.com/docs/styled-system/responsive-styles#the-object-syntax): `{ base: "100%", lg: "50%" }`

Reply:
woo, thanks for the tip!